README.md

```diff
@@ -163,38 +163,36 @@ password 1234....
 
 #### `get(variable_id)`
 
-Gets a variable value based on its ID. Variable is binary data that should be decoded to your
-system's encoding (e.g.
+Gets a variable value based on its ID. Variable is binary data that should be decoded to your system's encoding (e.g.
 `get(variable_id).decode('utf-8')`.
 
 #### `get_many(variable_id[,variable_id...])`
 
-Gets multiple variable values based on their IDs. Variables are returned in a dictionary that maps
-the variable name to its value.
+Gets multiple variable values based on their IDs. Variables are returned in a dictionary that maps the variable name to
```

Comment:
a dictionary? Is that the industry word? A dictionary is a מילון

Reply:
I found this: "The Dictionary class is the abstract parent of any class, such as Hashtable , which maps keys to values. Every key and every value is an object." Is this what a dictionary in our case?

Reply:
yep something like that